test/services/user/unit.test.js

```diff
@@ -1,5 +1,6 @@
 const assert = require('assert');
 const { expect } = require('chai');
+const { ObjectId } = require('mongoose').Types;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)